shannon/testnet/client.toml

```diff
@@ -0,0 +1,5 @@
+broadcast-mode = "sync"
+chain-id = "poktroll"
+keyring-backend = "test"
```

Comment:
Thoughts on renaming this to beta?

Sounds cool but I'm afraid it'll force us to change all our docs. If my assumption is correct, let's leave it as is.

Reply:
Changing keyring backend indeed will require us changing the docs (even replacing it with `file` like morse). I'm not sure if we should do it.
1. Regular users won't care as they use different wallets;
2. Production deployments will opt in to use a different backend: https://docs.cosmos.network/main/user/run-node/keyring#the-pass-backend